tests/TestCase.php

```diff
@@ -1,34 +1,46 @@
 <?php
 
-namespace PodPoint\KinesisLogger\Tests;
+namespace PodPoint\MonologKinesis\Tests;
 
-use Aws\Kinesis\KinesisClient;
 use Illuminate\Foundation\Application;
-use Mockery;
 use Orchestra\Testbench\TestCase as Orchestra;
-use PodPoint\KinesisLogger\KinesisMonologServiceProvider;
+use PodPoint\MonologKinesis\MonologKinesisServiceProvider;
 
 abstract class TestCase extends Orchestra
 {
     /**
      * Get package providers.
      *
-     * @param Application $app
-     *
+     * @param  Application  $app
      * @return array
      */
-    protected function getPackageProviders($app)
+    protected function getPackageProviders($app): array
     {
         return [
-            KinesisMonologServiceProvider::class,
+            MonologKinesisServiceProvider::class,
         ];
     }
 
     /**
-     * @return Mockery\MockInterface
+     * Define environment setup.
+     *
+     * @param  \Illuminate\Foundation\Application  $app
+     * @return void
      */
-    protected function getMockedKinesisClient()
+    protected function getEnvironmentSetUp($app): void
```

Comment:
```suggestion
    protected function setUpEnvironment($app): void
```
perhaps? - the `get` suggests it should return the environment setup

Reply:
This is coming from Orchestra itself
https://packages.tools/testbench/basic/define-environment.html#define-environment

Will change it to `defineEnvironment()`